downloads/pt_PT/github-git-cheat-sheet.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: cheat-sheet
+title: Folheto de ajuda para Git do GitHub (pt-PT)
+byline: Git é um sistema de controlo de versão distribuído open source que facilita ações com o GitHub no seu portátil ou desktop. Este folheto de ajuda resume instruções frequentemente usadas na linha de comando do Git para referência rápida.
+leadingpath: ../../
+---
+
+{% capture colOne %}
+## Instalar o Git
+
+Git é um sistema distribuído open source de controlo de versões que permite interagir com o GitHub no seu notebook ou desktop. Este folheto de ajuda resume as instruções frequentemente usadas na linha de comando do Git para referência rápida.
```

Comment:
this had been raised by @shana before and I lost the change (I know when 😞), good spot!